test/JDBC/expected/Test-sp_reset_connection.out

```diff
@@ -0,0 +1,109 @@
+-- 1. Test resets GUC variables
+SET lock_timeout 0;
+GO
+SELECT @@lock_timeout;
+GO
+~~START~~
+int
+0
+~~END~~
+
+EXEC sys.sp_reset_connection
+-- TODO: GUC is not resetting
+SELECT @@lock_timeout;
```

Comment:
I'm assuming the default value was 0.  How are we really testing the reset in this case?